interoperability-layer-openhim/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
         - prometheus-port=8080
 
   openhim-console:
-    image: ${OPENHIM_CONSOLE_VERSION}
+    image: ${OPENHIM_CONSOLE_IMAGE}
```

Comment:
I think there is already a env for defining the tag to use, but think the updated env naming is more in line with the others. We should then remove the older env:
https://github.com/jembi/platform/pull/289/files#diff-d1dbb584ce75d5c7c1a32820718b2e21ce4fa3fcce44cb1a473ba38a527293c2R38